### PR TITLE
{tools}[GCCcore/10.3.0] libtree v2.0.0

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/l/libtree/libtree-2.0.0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+    ('cxxopts', '3.0.0'),
+    ('ELFIO', '3.9'),
+    ('termcolor', '2.0.0'),
+]
+
+sanity_check_paths = {
+    'files': ["bin/%(name)s"],
+    'dirs': ['bin', 'share']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/l/libtree/libtree-2.0.0-GCCcore-10.3.0.eb:31]:
```suggestion
    'dirs': ['share']
```
(no need to check dir `bin` if already checking file inside it)

**Review by migueldiascosta — DISMISSED** [review]

File: easybuild/easyconfigs/l/libtree/libtree-2.0.0-GCCcore-10.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+sanity_check_commands = ["libtree --version"]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/l/libtree/libtree-2.0.0-GCCcore-10.3.0.eb:34]: an interesting alternative would be `libtree -ap %(installdir)s/bin/libtree` :p

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2624.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/3598f2ab36c9c4079d47f1bc49513edb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3518.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/ce6ccc0f45681fb40e64001fba7e6215 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3518.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/12e51e6665028866d900f03ca6d65ec8 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3169.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/94f58a49fce96e18b1b24d2c948c638d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**